spec.md

```diff
@@ -1897,6 +1940,8 @@ This RPC will be called by the CO to delete a snapshot.
 This operation MUST be idempotent.
 If a snapshot corresponding to the specified `snapshot_id` does not exist or the artifacts associated with the snapshot do not exist anymore, the Plugin MUST reply `0 OK`.
 
+This RPC MUST not be called if the `group_snapshot_id` field is not empty as individual snapshots that are part of a group snapshot will be deleted together with the group snapshot when the group snapshot is deleted.
```

Comment:
How about this:

Snapshots that are members of a group snapshot will be deleted when the group snapshot is deleted. The CO SHOULD NOT specify a `snapshot_id` for a snapshot that was created as part of a group snapshot, i.e. if the snapshot had a non-empty `group_snapshot_id` field at creation time. The SP MAY refuse to delete such snapshots with this RPC call and return an error instead.

Reply:
Done